src/core/objects/PDFArray.ts

```diff
@@ -47,6 +47,17 @@ class PDFArray extends PDFObject {
     return this.array[index];
   }
 
+  lookupElements(): Array<PDFObject | undefined>;
+  lookupElements(type: typeof PDFDict): PDFDict[];
+
+  lookupElements(type?: any) {
+    const values = new Array(this.size());
+    for (let idx = 0, len = this.size(); idx < len; idx++) {
+      values[idx] = this.context.lookup(this.get(idx), type);
```

Comment:
This is pedantic, but I think you can simplify this to the following:
```suggestion
      values[idx] = this.lookup(idx, type);
```